config/internal/common/policy.yaml.tmpl

```diff
@@ -45,8 +45,24 @@ spec:
             matchLabels:
               app: ds-pipeline-scheduledworkflow-{{.Name}}
               component: data-science-pipelines
+        - podSelector:
+            matchLabels:
+              app: ds-pipeline-metadata-envoy-{{.Name}}
+              component: data-science-pipelines
+        - podSelector:
+            matchLabels:
+              app: ds-pipeline-metadata-grpc-{{.Name}}
+              component: data-science-pipelines
+        - podSelector:
+            matchLabels:
+              app: ds-pipeline-metadata-writer-{{.Name}}
+              component: data-science-pipelines
       ports:
         - protocol: TCP
           port: 8888
         - protocol: TCP
           port: 8887
+        - protocol: TCP
+          port: 9090
+        - protocol: TCP
```

Comment:
this policy of for dsp apiserver, and that pod does not have these ports, so remove these additions

Reply:
Removed these extra ports.